src/checks/y_check_constants_interface.clas.abap

```diff
@@ -62,16 +55,16 @@ CLASS Y_CHECK_CONSTANTS_INTERFACE IMPLEMENTATION.
     AND token <> 'OF'.
       has_something_else = abap_true.
     ENDIF.
+    IF token = 'CONSTANTS'.
+      has_at_least_one_constant = abap_true.
+    ENDIF.
   ENDMETHOD.
 
 
   METHOD check_result.
-    CHECK has_something_else = abap_false.
-
+    CHECK has_something_else = abap_false AND has_at_least_one_constant = abap_true. " #561
```

Comment:
Please don't add "edit history" comments like this - the git history in the repository will already show the pull request number, there is no need to duplicate it in the code.